Sources/Brave/Frontend/Browser/New Tab Page/Sections/BraveNewsSectionProvider.swift

```diff
@@ -441,8 +462,9 @@ class BraveNewsSectionProvider: NSObject, NTPObservableSectionProvider {
       itemActionHandler(.toggledSource, context)
     }
 
-    return .init { [weak self] index -> UIMenu? in
+    let newItemAction: FeedItemMenu = .init { [weak self] index -> UIMenu? in
       guard let self = self else { return nil }
+      
```

Comment:
Can be reverted along with the part below

Reply:
Fixed at https://github.com/brave/brave-ios/pull/7589/commits/e5ca33debfb25eca4c069bdc2c2670c82603fb64